kani-compiler/src/kani_compiler.rs

```diff
@@ -140,7 +140,11 @@ impl Callbacks for KaniCompiler {
                 .set_check_assertion_reachability(matches.get_flag(parser::ASSERTION_REACH_CHECKS));
             queries.set_output_pretty_json(matches.get_flag(parser::PRETTY_OUTPUT_FILES));
             queries.set_ignore_global_asm(matches.get_flag(parser::IGNORE_GLOBAL_ASM));
-            queries.set_write_goto_binary(matches.get_flag(parser::WRITE_GOTO_BINARY));
+            if cfg!(feature = "write_json_symtab") {
```

Comment:
Nit: might be simpler to write as:
```rust
queries.set_write_json_symtab(cfg!(feature = "write_json_symtab") || matches.get_flag(parser::WRITE_JSON_SYMTAB));
```
but not a biggie.

Reply:
fixed